src/test/java/com/keeper/homepage/domain/library/application/BookServiceTest.java

```diff
@@ -144,4 +149,78 @@ void setUp() {
       assertThrows(BusinessException.class, () -> bookService.requestBorrow(member, book.getId()));
     }
   }
+
+  @Nested
+  @DisplayName("도서 반납 요청")
+  class requestReturn {
+
+    private Member member;
+    private Book book;
+
+    @BeforeEach
+    void setUp() {
+      member = memberTestHelper.generate();
+      book = bookTestHelper.generate();
+    }
+
+    @Test
+    @DisplayName("도서 반납 요청 시 lastRequestDate가 갱신되어야 한다.")
+    public void 도서_반납_요청_시_lastRequestDate가_갱신되어야_한다() throws Exception {
+      BookBorrowInfo bookBorrowInfo = bookBorrowInfoTestHelper.builder()
+          .member(member)
+          .book(book)
+          .borrowStatus(getBookBorrowStatusBy(대출승인))
+          .build();
+
+      long borrowId = bookBorrowInfo.getId();
+
+      LocalDateTime now = LocalDateTime.now();
+      bookService.requestReturn(member, borrowId);
+      BookBorrowInfo updateTime = bookBorrowInfoRepository.findById(borrowId).orElse(null);
+
+      assertThat(updateTime).isNotNull();
+      assertNotEquals(now, updateTime.getLastRequestDate());
+    }
+
+    @Test
+    @DisplayName("도서 반납 요청 시 도서 상태가 갱신되어야 한다.")
+    public void 도서_반납_요청_시_도서_상태가_갱신되어야_한다() throws Exception {
+      BookBorrowInfo bookBorrowInfo = bookBorrowInfoTestHelper.builder()
+          .member(member)
+          .book(book)
+          .borrowStatus(getBookBorrowStatusBy(대출승인))
+          .build();
+
+      long borrowId = bookBorrowInfo.getId();
+
+      bookService.requestReturn(member, borrowId);
+      BookBorrowInfo borrowInfo = bookBorrowInfoRepository.findById(borrowId).orElse(null);
+
+      assertThat(borrowInfo).isNotNull();
```

Comment:
```suggestion
      BookBorrowInfo borrowInfo = bookBorrowInfoRepository.findById(borrowId).orElseThrow();
```

이건 뭐가 더 낫다할 순 없지만, 개인적으론 `isNotNull()`을 따로 검증하기보다 바로 `throw`를 던지면 test가 깨진 곳을 바로 찾아줘서 고치기 쉬운 것 같아요~ @JJimini 님처럼 해놓으면 `isNotNull`인걸 확인하고 위 코드를 훑어보면서 어디서 `null`이 들어왔는지 찾아야 할 것 같아서요~~

Reply:
그렇네요...! 하나 또 배워갑니다!👍👍